### DISCUSSION TITLE
Is Pomotroid abandoned?

### DISCUSSION THREAD

**Discussion by lpepen-re** [Splode/pomotroid#266] [Q&A] [answered]
I just found this great project, i love its visuals simplicity, but i've notice the last code merge was around 2 years go, there are over a 100 PRs from Dependabot and the community which have gone ignored. Just wondering if this project is abandoned or not. if so Achieving the Repo would make it explicity clear that it is.

**Comment by zalexk** [comment]
I do think it's really be abandoned (though there isn't any offical notice)

**Reply by IordanisKokk** [reply]
Yeah I also think so, It does work great though, but I would like to add a feature to it, although there is no hope of it getting on the actual app.

**Comment by moatmai** [comment, marked as answer]
It's **ALIVE**. Version 1.10 was released just a couple of days ago.

**Reply by lpepen-re** [reply]
Thank you @Splode specially the complete rewrite to Svelte + Tauri ♥️
